viam_cartographer.go

```diff
@@ -127,9 +127,11 @@ func initSensorProcess(cancelCtx context.Context, cartoSvc *CartographerService)
 	}
 
 	cartoSvc.sensorProcessWorkers.Add(1)
+	isOnline := cartoSvc.lidarDataRateMsec != 0
```

Comment:
Since the data rate msec is already being passed via spConfig, can we just check the data rate msec directly in Start?

Then we don't have to worry about the name (isOnline vs. live), and can just add a comment that when the data rate msec is 0, we assume the user wants to be in "offline" mode and ensure every scan gets processed by cartographer

Reply:
ah got it! changing